receiver/prometheusreceiver/internal/internal_test.go

```diff
@@ -17,7 +17,9 @@ package internal
 import (
 	"context"
 	"errors"
+
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
```

Comment:
nit: move this import to the group on the bottom

Reply:
Fixed.